### DISCUSSION TITLE
Flagged

### DISCUSSION THREAD

**Discussion by kennyponde** [community/community#46095] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

hello, can someone please explain to me why my account is flagged? I brand new with GitHub but I don't understand why my account is flagged

**Comment by cehfisher** [comment]
Hi @kennyponde - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to a different category? Thanks!
